src/common/components/NotificationOverlay.tsx

```diff
@@ -0,0 +1,97 @@
+import { FC, ReactNode } from 'react';
```

Comment:
- The original notification Ui goes back to blocks and we will be directly using the notification component from uiweb for the channel notification use case.